ansible_runner/runner.py

```diff
@@ -259,32 +258,26 @@ def host_events(self, host):
         return all_host_events
 
     @classmethod
-    def handle_termination(cls, pid, args, proot_cmd, is_cancel=True):
+    def handle_termination(cls, pid, is_cancel=True):
         '''
         Internal method to terminate a subprocess spawned by `pexpect` representing an invocation of runner.
 
         :param pid:       the process id of the running the job.
-        :param args:      the args for the job, i.e., ['ansible-playbook', 'abc.yml']
-        :param proot_cmd  the command used to isolate processes i.e., `bwrap`
         :param is_cancel: flag showing whether this termination is caused by
                           instance's cancel_flag.
         '''
         try:
-            if proot_cmd and proot_cmd in ' '.join(args):
-                if not psutil:
-                    os.kill(pid, signal.SIGKILL)
-                else:
-                    try:
-                        main_proc = psutil.Process(pid=pid)
-                        child_procs = main_proc.children(recursive=True)
-                        for child_proc in child_procs:
-                            os.kill(child_proc.pid, signal.SIGKILL)
-                        os.kill(main_proc.pid, signal.SIGKILL)
-                    except (TypeError, psutil.Error):
-                        os.kill(pid, signal.SIGKILL)
+            if not psutil:
+                os.kill(pid, signal.SIGKILL)
```

Comment:
I don't think this psutil check is even necessary anymore. In the old AWX codebase it was originally conditionally imported because it wasn't available in a good form on py2.6. This path can be removed entirely.